tests/mocha/shortcut_registry_test.js

```diff
@@ -25,14 +25,14 @@ suite('Keyboard Shortcut Registry Test', function() {
     test('Registering a shortcut', function() {
       const testShortcut = {'name': 'test_shortcut'};
       this.registry.register(testShortcut, true);
-      const shortcut = this.registry.registry_['test_shortcut'];
```

Comment:
these tests should probably use the public APIs instead of messing with the internal object :/ but you don't have to fix that rn

Reply:
Went ahead and fixed this, and various other issues where there were extra/missing params/wrong types/etc.